src/System.Management.Automation/utils/StructuredTraceSource.cs

```diff
@@ -290,7 +290,6 @@ internal PSTraceSource(string fullName, string name, string description, bool tr
                 FullName = fullName;
                 _name = name;
 
-#if !CORECLR // TraceSource.Attributes is not in CoreCLR
                 string tracingEnvVar = Environment.GetEnvironmentVariable("MshEnableTrace");
```

Comment:
Do we want to depend on an environment variable?

Reply:
In general, I don't want PowerShell to depend on environment variables.  We should move this to the startup json file.  I'll add a TODO comment.